<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbUcitajPesmu">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>999</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ucitaj pesmu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSacuvajIzmene">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>740</y>
      <width>999</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sacuvaj izmene</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>441</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="leIzvodjac"/>
     </item>
     <item>
      <widget class="QLineEdit" name="leNaziv"/>
     </item>
     <item>
      <widget class="QLineEdit" name="leAlbum"/>
     </item>
     <item>
      <widget class="QLineEdit" name="leRejting"/>
     </item>
     <item>
      <widget class="QLineEdit" name="leGodina"/>
     </item>
     <item>
      <widget class="QLineEdit" name="leGostujuci"/>
     </item>
     <item>
      <widget class="QLineEdit" name="leZanrovi"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>441</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbIzvodjac">
       <property name="text">
        <string>Izvodjc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbNaziv">
       <property name="text">
        <string>Naziv</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbAlbum">
       <property name="text">
        <string>Album</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbRejting">
       <property name="text">
        <string>Rejting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbGodina">
       <property name="text">
        <string>Godina</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbGostujuci">
       <property name="text">
        <string>Gostujuci Izvodjaci</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbZanrovi">
       <property name="text">
        <string>Zanrovi</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuZadatak_6_1">
    <property name="title">
     <string>Zadatak 6.1</string>
    </property>
   </widget>
   <addaction name="menuZadatak_6_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
